### ISSUE TITLE
Recording of Manfed Wuhreres talk

### ISSUE THREAD

**Issue by ReneRanzinger** [glygener/glygen-issues#641]
Hi @jeet-vora who will be working on this. You or Junyue?

**Comment by jeet-vora** [comment]
@ReneRanzinger I have worked on it to test the software.
The recording is here https://drive.google.com/drive/folders/1m0Vls87qXVd5qxxt9kiJKrTnQCE2Brs6
Let me know if it works, then I can ask Jingyue to work on others.

**Comment by ReneRanzinger** [comment]
I uploading the video. I will need:
- thumbnail in video resolution (1920x1080): Will be shown a pictogram in youtube searches
- chapters in format
```
00:00:00 GLIC seminar series
00:03:00 Introduction to Glycan Arrays
00:19:40 MIRAGE
```

I have the keywords:
- GlyGen
- glycomics
- mass spectrometry
- carbohydrates

Let me know if you want to add others.

**Comment by jeet-vora** [comment]
<ins>**Thumbnails**</ins>
Thumbnail PNG - 
![GlyGen_Webinar_Series_Manfred_Wuhrer_Aug22_2023_thumbnail](https://github.com/glygener/glygen-issues/assets/25825266/223873df-a74a-49b2-a15c-97ecb4142551)

Thumbnail MP4 (720p, 1080p not available in free version) - 
https://github.com/glygener/glygen-issues/assets/25825266/a0d95aa2-0260-4264-b3fe-6fa8f207d636

<ins>**Chapters**</ins>

**Introduction:** 00:00:00 - 00:01:39
**Isomers of Glycan, cell-tissue specificity:** 00:01:40 - 00:04:06
**Chromatographic Profiles of CRC:** 00:04:07 - 00:08:28
**Protein-Site Glycosylation:** 00:08:29 - 00:27:57
**N-Glycosylation of Blood Proteins**: 00:27:58 - 00:35:21
**Conclusion:** 00:35:22 - 00:37:08
**Acknowledgment:** 00:37:09 - 00:37:49

<ins>**Keywords**</ins>

glycomics
mass-spectrometry
IgG
colorectal cancer
blood proteins
glycosylation

**Comment by ReneRanzinger** [comment]
@jeet-vora I made the video public (https://www.youtube.com/watch?v=F-B6kltstKM) you can let Manfred know.

@jeet-vora and @JingyueWu YouTube added a new feature. If you go to the video and click in the description section on "show more" you will see all my information and a new chapter section. Unfortunately, that new features uses the same Thumbnail for all chapters. It seems there is way to have individual thumbnails per chapter (https://www.youtube.com/watch?v=ySqnRRNQyhk) for the next videos we may want to look into this.

**Comment by jeet-vora** [comment]
Yes we can add the thumbnail to individual chapters maybe screen shot or some different colored page with the chapter title.

**TASK: Add slide to Wiki and YouTube.**
